katran/lib/testing/katran_tester.cpp

```diff
@@ -246,6 +246,12 @@ void testLbCounters(katran::KatranLb& lb) {
       LOG(INFO) << "Expected to be incorrect w/ non default build flags";
     }
   }
+  auto lb_stats = lb.getKatranLbStats();
+  if (lb_stats.bpfCalls != 961346 || lb_stats.bpfFailedCalls != 0) {
```

Comment:
lol magic number.  I assume this just happens to be the number of bpf calls in test currently?  Is there a way to make this a constant or defined in terms of other things here?  Otherwise devs may end up chasing this number when something changes.

Reply:
it is just collections of uint64s. copy of em would be faster than point dereference. and i cannot imagine situation when someone would want to have a refernce instead of copy (because right now we would need to have a lock only when we run getKatranLbStats, with reference any manipulation (read/write etc) would require this lock

as for magic number:
- yes, this is how many calls are being done w/ default ch ring size and config. there are pretty few other counters like this and tests which are related on default ch ring size and default salts for jhash.

- not related anymore as we have deleted this bpfCalls counter